client/rpcserver/handlers.go

```diff
@@ -452,4 +461,71 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	exchangesRoute: {
+		pwArgsShort: ``,
+		argsShort:   ``,
+		cmdSummary:  `List assets supported by all registered DEX servers their and active trades.`,
+		pwArgsLong:  ``,
+		argsLong:    ``,
+		returns: `Returns:
+    map: The exchanges result.
+    {
+      "[DEX url] (map): {
+        "url" (string): The DEX URL.
+        "markets" (map): {
+          "[assetID-assetID]" (map): {
+            "name" (string): The market name. e.g. "dcr_btc".
+            "baseid" (int): The base assetID
+            "basesymbol" (string): The base assetID symbol.
+            "quoteid" (int): The quote assetID.
+            "quotesymbol" (string): The quote assetID symbol,
+            "epochlen" (int): Duration of a trade window in milliseconds.
+            "startepoch" (int): Unix time of start of the last epoch in seconds
+	      since 00:00:00 Jan 1 1970.
```

Comment:
Unfortunately, we need to mention milliseconds if we call it "Unix time", since Unix time is in seconds by definition.